packages/web3-errors/test/unit/__snapshots__/errors.test.ts.snap

```diff
@@ -156,6 +156,17 @@ exports[`errors Eip838ExecutionError should get the data from error.data.origina
 }
 `;
 
+exports[`errors Eip838ExecutionError should return correctly when data is undefined 1`] = `
+{
+  "cause": undefined,
+  "code": undefined,
+  "data": undefined,
```

Comment:
The code was already working when `data` is `undefined`. But it does not work when data is `null`. So, could you please update the tests?
Thanks,

Reply:
ok I will look into it and fix.

Reply:
Hi @Alfxjx ,
Just to clarify that your code looks great! And what I proposed to update is only your tests. That is to test with `null` instead of `undefined`. Please, let me know if you need any help to finish this.
Thanks,